tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -3015,9 +3052,24 @@ public function userShouldBeAbleToAcceptShareOfferedBy(
 		string $offeredBy
 	) {
 		if ($action === 'accept') {
-			$this->userHasAcceptedThePendingShareOfferedBy($user, $share, $offeredBy);
+			$response = $this->reactToShareOfferedBy($user, 'accepts', $share, $offeredBy, 'pending');
+			$this->setResponse($response);
```

Comment:
let's not use `setResponse` in Then steps as well (it will give trouble when we use chain of Then steps)
```suggestion
			$this->setResponse($response);
```